### DISCUSSION TITLE
Xiaomi Vibration Sensor

### DISCUSSION THREAD

**Discussion by KayGt3** [indomus/forum#1859] [GENERALE - Discussioni generiche]
Stavo pensando di utilizzare il sensore di [vibrazione](https://www.zigbee2mqtt.io/devices/DJT11LM.html) della Xiaomi come antifurto per le tapparelle.

Qualcuno ha già adottato questa soluzione? Nel caso sapere consigliarmi altre soluzioni?

Grazie!

**Comment by snakuzzo** [comment]
Premesso che non ho questi sensori...ma rischieresti una marea di falsi allarmi.
Secondo me ti basterebbe un sensore di presenza

**Reply by KayGt3** [reply]
Ho già dei sensori sulle finestre della Xiaomi, ovviamente in estate le finestre rimangono aperte e stavo cercavo quindi una soluzione per le tapparelle (purtroppo non posso mettere i sensori porte/finestre Xiaomi sulle tapparelle).

Sai se esistono dei sensori di movimento che abbiamo il fascio piatto? Cosi da limitare il raggio solo nella zona della tapparella.
